common/web/types/src/main.ts

```diff
@@ -29,4 +29,8 @@ export * as KPJ from './kpj/kpj-file.js';
 export { KPJFileReader } from './kpj/kpj-file-reader.js';
 export { KeymanDeveloperProject } from './kpj/keyman-developer-project.js';
 
-export * as util from './util/util.js';
\ No newline at end of file
+export * as KpsFile from './package/kps-file.js';
```

Comment:
I get why `KmpJsonFile` fits decently in common/web/types - good old `kmp.json` is used in the mobile apps, after all.

But... why does the `KpsFile` definition "belong better now in common/web/types"?  Isn't that only used in Developer stuff, alongside KPJ stuff - which is _not_ moved by this PR?

Reply:
Yeah, we could have a developer/src/common/web/types as well for .kps, .kvks, etc. For now I've opted to just have the one module. Happy to have that discussion again before we release 17.0.

I'm moving all the "purer" file format interfaces and readers/writers slowly over into common/web/types. It helps keep it compartmentalised and maintain the relationship between file types. There is a bit of a fuzzy boundary, of course; some of the trivial 'compile' phases end up there too -- e.g. kvks into kvk. More complex compilation stays under Developer. 

You'll see .kvks, .keyman-touch-layout also in common/web/types. Also, KPJ is in common/web/types already. See kpj-file.ts, kpj-file-reader.ts.

Note, kmp.json is used in a lot more than just the mobile apps!